torch/distributed/pipelining/schedules.py

```diff
@@ -2218,7 +2263,7 @@ def _ready_to_schedule(action: Optional[_Action]) -> bool:
                 ):
                     return True
             return False
-        elif action.computation_type == B:
+        elif action.computation_type in (B, BW):
```

Comment:
From what I could tell, all the `BW` changes are changes the the schedule plan side, but don't see need the logic where `BW` actually gets executed? As in where is the logic that checks:
```python
if computation_type == "BW":
    # perform backward_one_chunk(), etc.
```

Reply:
fixed